week-1/Homework/mandatory/4-tax.js

```diff
@@ -5,7 +5,12 @@
   Sales tax is 20% of the price of the product
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(price) {
+  const salesTax= price*20/100;
```

Comment:
All the symbols are next to each other here, think about adding spaces in there to make it tidier:
```suggestion
  const salesTax = price * 20/ 100;
```